velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -462,3 +462,78 @@ TEST_F(DecimalArithmeticTest, negate) {
            DecimalUtil::kLongDecimalMin},
           DECIMAL(38, 19))});
 }
+
+TEST_F(DecimalArithmeticTest, between) {
+  // DecimalBetweenFunction handles both short decimals and long decimals the
+  // same way, it is agnostic to the underlying value type.
+  //
+  // Below we only verify short decimal vectors.
```

Comment:
Can we choose long decimal vectors instead since long decimals throw the most curveballs?

Reply:
Actually, let me add tests for both decimal types.